### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Gracetemitope**
- Initialized React App
- Set up React Router and React Redux
- Structure project folder using feature folder approach
- Displayed List of Books

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="d-flex">
+      <input
+        type="text"
+        className="form-control ml-3"
+        placeholder="Book Title"
+        name="title"
+      />
+      <select name="category" className="form-control">
+      <option value="volvo">category</option>
+      <option value="saab">Biography</option>
```
Comment (Davidosky007) [inline src/components/AddBook.js:17]: * Kindly add a button to submit the form the button wouldn't add the book yet, this is just a placeholder for now
* Refer to project sneak peek 
image

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `yarn build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
```
Comment (Davidosky007) [inline README.md]: * Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project

File: src/components/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const BookCategory = () => {
+  return (
+    <div>
+      <h1 className="text-center">Under Construction</h1>
+    </div>
```
Comment (Davidosky007) [inline src/components/redux/categories/categories.js]: * Kindly fix your redux folder as it should be inside `src` (src/redux/books/books.js) and not (src/components/redux/books/books.js)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Gracetemitope 👋, 

Good job so far! 💪 
* The repo looks professional.✔️
* Git/GitHub flow best practices have been followed✔️
* There are no linter errors✔️
* You have descriptive PR✔️

## There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Davidosky007** [comment]
## Hi @Gracetemitope 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Gracetemitope** [comment]
Dear Reviewer, I'm repeating the week. This PR has  been approved previously. I only need to submit the task. Thanks

**Review by oxenprogrammer — COMMENTED** [review]
Hi @Gracetemitope , 
This project was already approved. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**